apps/tutorial_servo_control_can/TutorialServo.cpp

```diff
@@ -0,0 +1,28 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin) 
+	:_servoPwmOut(servoPin), _minPulseWidthInMs(1), _maxPulseWidthInMs(2), _servoRangeInDegrees(180)
+{
+	_minPulseWidthInMs = _minPulseWidthInMs / 1000;

```

Comment:
why divide by 1000?

Reply:
the user is entering the seconds in milliseconds. if the user enters 1, then it should be 0.001. hence I divide by 1000 to convert to seconds even though the user enters it in milliseconds

Reply:
should use std::chrono